doc/source/tutorial/stats/sampling_pinv.rst

```diff
@@ -0,0 +1,271 @@
+.. _sampling-pinv:
+
+Polynomial interpolation based INVersion of CDF (PINV)
+======================================================
+
+.. currentmodule:: scipy.stats
+
+* Required: PDF
+* Optional: CDF, mode, center
+* Speed:
+
+  * Set-up: (very) slow
+  * Sampling: (very) fast
+
+Polynomial interpolation based INVersion of CDF (PINV) is an inversion method
+that only requires the density function to sample from a distribution. It is
+based on Polynomial interpolation of the PPF and Gauss-Lobatto integration
+of the PDF. It provides control over the interpolation error and integration
+error. Its primary purpose is to provide very fast sampling which is nearly
+the same for any given distribution at the cost of moderate to slow setup
+time. It is the fastest known inversion method for the fixed-parameter case.
+
+The inversion method is the simplest and most flexible to sample nonuniform
```

Comment:
This general introduction could also be moved to the main tutorial on sampling. For example, it is also relevant for HINV.

Reply:
I was thinking of expanding the main tutorials to explain each of the methods separately instead of explaining them in a single paragraph. Let me know what you think about that. I can open a PR in parallel where we can track those changes.

Reply:
Agree, we can keep it as it is and expand the main tutorial as a follow-up